src/config/config.json

```diff
@@ -1,21 +1,21 @@
 {
   "development": {
     "username": "postgres",
-    "password": "test123",
-    "database": "developmentdb",
+    "password": "nmjules77",
+    "database": "teamdb",
     "host": "127.0.0.1",
```

Comment:
You need to use environment variables. So please consider use the javascript configuration file instead of json file.